codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java

```diff
@@ -57,15 +57,25 @@
      * @param server {@code true} this is a server
      */
     public Http2FrameCodec(boolean server) {
-        this(server, new DefaultHttp2FrameWriter());
+        this(server, true);
+    }
+
+    /**
+     * Construct a new handler.
+     *
+     * @param server {@code true} this is a server
+     */
+    public Http2FrameCodec(boolean server, boolean verbose) {
+        this(server, verbose, new DefaultHttp2FrameWriter());
     }
 
     // Visible for testing
-    Http2FrameCodec(boolean server, Http2FrameWriter frameWriter) {
+    Http2FrameCodec(boolean server, boolean verbose, Http2FrameWriter frameWriter) {
         Http2Connection connection = new DefaultHttp2Connection(server);
-        frameWriter = new Http2OutboundFrameLogger(frameWriter, HTTP2_FRAME_LOGGER);
+        frameWriter = verbose ? new Http2OutboundFrameLogger(frameWriter, HTTP2_FRAME_LOGGER) : frameWriter;
```

Comment:
changed it to take a frame logger as an argument, so we don't have to worry about this anymore.